src/NHibernate.Test/TypesTest/DateTypeTest.cs

```diff
@@ -55,36 +56,30 @@ public void ReadWriteNormal()
 		{
 			var expected = DateTime.Today.Date;
```

Comment:
Do not need to call `.Date` on `.Today` as it is already a date

Reply:
Sure I could have cleaned that, just had refactored previous code in a common method without changing the logic.